### PR TITLE
DHFPROD-7829: Component Replacement: Pagination

### PR DESCRIPTION
**Pull request by iveyMarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/relatedEntityMapping.spec.tsx [hunk 1]
```diff
@@ -264,7 +264,7 @@ describe("Mapping", () => {
     mappingStepDetail.getRelatedEntityFromList("Relation (relatedTo Person)");
     cy.get("#entityContainer").scrollTo("bottom",  {ensureScrollable: false});
     mappingStepDetail.entityTitle("Person").should("not.be.visible");
-    browsePage.getPaginationPageSizeOptions().then(attr => {
+    cy.get(".ant-pagination-options .ant-select-selection-selected-value").then(attr => {
```
Comment (csilva-ml) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/relatedEntityMapping.spec.tsx:267]: I suggest not doing any "cy.get" inside spec files, is there a page object you can place it in?

Comment (iveyMarklogic) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/relatedEntityMapping.spec.tsx]: @csilva-ml  this was solved

**Review by mrodrigu-marklogic — DISMISSED** [review]
LGTM

**Review by briantang — COMMENTED** [review]
Functionality looks good. Buttons look interesting :) 
@iveyMarklogic Did UX check this? Maybe they were OOO. If so, I think it's fine to move this along and check when they return.


Screen Shot 2021-11-23 at 9 16 02 PM

**Comment by MSBrignoni** [comment]
@iveyMarklogic the behaviors and styles looks great. Perhaps could it be possible to avoid the button click event on the same page I'm on and maybe paint the chosen paging options in bold as on nightly? 
image

**Comment by iveyMarklogic** [comment]
@MSBrignoni both changes were added

**Review by federicoRufrancosMarklogic — APPROVED** [review]

**Review by MSBrignoni — APPROVED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Pull request merged**